### PR TITLE
ceph-volume: batch bluestore fix create_lvs call

### PR DESCRIPTION
**Pull request by jan--f**
Fixes: https://tracker.ceph.com/issues/43844

Signed-off-by: Jan Fajerski <jfajerski@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by jan--f** [comment]
jenkins test ceph-volume tox

File: src/ceph-volume/ceph_volume/api/lvm.py [hunk 1]
```diff
@@ -1333,7 +1333,7 @@ def create_lvs(volume_group, parts=None, size=None, name_prefix='ceph-lv'):
         size = sizing['sizes']
         extents = sizing['extents']
         lvs.append(
-            create_lv(name_prefix, uuid.uuid4(), vg=volume_group.name, extents=extents, tags=tags)
```
Comment (sebastian-philipp) [inline src/ceph-volume/ceph_volume/api/lvm.py]: Looks like a trivial type error. As you don't want to add Py2 type hints, when do you expect c-v will be py3 on master?

File: src/ceph-volume/ceph_volume/devices/lvm/strategies/bluestore.py [hunk 2]
```diff
@@ -103,8 +103,7 @@ def execute(self):
 
         # create the lvs from the vgs captured in the beginning
         for create in osd_vgs.values():
-            block_uuid = system.generate_uuid()
-            lvs = lvm.create_lvs('osd-data', block_uuid, vg=create['vg'], parts=create['parts'])
+            lvs = lvm.create_lvs(create['vg'], parts=create['parts'], name_prefix='osd-data')
```
Comment (sebastian-philipp) [inline src/ceph-volume/ceph_volume/devices/lvm/strategies/bluestore.py]: in theory type checkers should be able to find these errors without any type hints. Unfortunately mypy is not there yet.

File: src/ceph-volume/ceph_volume/api/lvm.py (hunk 1 above)
Comment (jan--f) [reply src/ceph-volume/ceph_volume/api/lvm.py]: whenever the last py2 release (nautilus iiuc) goes out of support. So after the P release, or Q?

**Comment by jan--f** [comment]
@dsavineau Guess I'll hear from you if this doesn't fix your issue.

**Comment by dsavineau** [comment]
@jan--f sure I'll keep you updated if something still doesn't work.

Thanks for the quick fix.

**Pull request merged**